digital-workplace-webapps/src/main/webapp/vue-app/topBar/notification/notificationsAPI.js

```diff
@@ -18,4 +18,30 @@
         method: 'PATCH',
         body: operation
     });
-}
\ No newline at end of file
+}
+
+  export function getCometdSettings() {
```

Comment:
```suggestion
  export function getUserToken() {
```
Should be called directly by initCometd not from .vue

Reply:
Done!